### ISSUE TITLE
Zero Avergae Precision and No Predictions

### ISSUE THREAD

**Issue by DohaRagab1** [benihime91/pytorch_retinanet#10]
I am following the demo notebook and applying it to my own dataset. When I train the model, sometimes it gives nan losses for every epoch. When I rebuild the model and rerun the cell -with the same parameters- it reads values.
Then it always gives zero average precision, although it may read loss values for the epochs. It has no predictions nor bounding boxes.
I can't figure out the solution.
